rel/overlay/bin/couchup

```diff
@@ -172,6 +172,25 @@ def _put_filter(args, db=None):
             print(exc.response.text)
             exit(1)
 
+def _do_security(args, db=None):
+    """Copies the _security object from source to target DB."""
+    try:
+        req = requests.get(
+            'http://127.0.0.1:{}/{}/_security'.format(
+                args['local_port'], db),
+                auth=args['creds'])
+        req.raise_for_status()
+        security_doc = _tojson(req)
+        req = requests.put(
+            'http://127.0.0.1:{}/{}/_security'.format(
+                args['clustered_port'], db),
+                data=json.dumps(security_doc),
```

Comment:
Do we need to set `headers = {'Content-type': 'application/json'}`

Reply:
@nickva requests automatically does this for us :heart: I actually tested the code.